src/components/molecules/CarouselText/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './style.css';
+import Title from '../../atoms/Title/index';
+
+function CarouselText() {
+    return (
+        <div className="carousel__text">
+            <Title className="title__carousel" title='Made with love' />
```

Comment:
wrong bem naming

Reply:
Changed names, but I need to think about how to properly name the components.